src/ionosphere/waccmx/ionosphere_interface.F90

```diff
@@ -392,27 +393,29 @@ subroutine ionosphere_run1(pbuf2d)
       real(r8), pointer :: pbuf_amie_kevg(:) ! Pointer to AMIE mean energy in pbuf
 
       integer :: ncol
-      real(r8), allocatable :: amie_efxg(:) ! energy flux from AMIE
-      real(r8), allocatable :: amie_kevg(:) ! characteristic mean energy from AMIE
+      real(r8), pointer :: amie_efxg(:) ! energy flux from AMIE
+      real(r8), pointer :: amie_kevg(:) ! characteristic mean energy from AMIE
```

Comment:
Are pointers better than allocatables?

Reply:
No. I needed pointers so that I could pass the data into the gridded component without copying all the data into import state fields (all those pointers at the top of edyn_grid_comp). This was a workaround to having to copy all the data into `ESMF_Field`s which would then be added to the import state.